spec/integration/posts_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+RSpec.describe 'Post', type: :feature do
```

Comment:
- [ ] 38 tests out of 50 examples, are failing. 
<img width="828" alt="Screenshot 2022-09-07 at 11 11 17" src="https://user-images.githubusercontent.com/77038610/188825974-f167f1ac-e5f5-419d-8811-54d4af189774.png">

I checked the reason of couple of them. In some specs the user is not being created, and users it is nil, so the specs fail. This is happening because of validation you have in User model:
`validates :posts_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }`

and the error is: 

` Validation failed: Posts counter is not a number (ActiveRecord::RecordInvalid)`

Please make sure that all specs are passing, perhaps you will need to debug your code, you can use `byebug` gem for that.